ui/app/adapters/secret-engine.js

```diff
@@ -132,7 +132,11 @@ export default ApplicationAdapter.extend({
 
   saveZeroAddressConfig(store, type, snapshot) {
     const path = encodePath(snapshot.id);
-    const roles = store.peekAll('role-ssh').filterBy('zeroAddress').mapBy('id').join(',');
+    const roles = store
+      .peekAll('role-ssh')
+      .filter((role) => role.zeroAddress === true)
```

Comment:
```suggestion
      .filter((role) => role.zeroAddress)
```
I think should work. I tried this in the console:
```
var arr = [{ selected: true }, { selected: true }, { selected: false }]
arr.filter(a => a.selected) // => returns [{ selected: true }, { selected: true }]
```

Reply:
I was being overly cautious about making sure it's a boolean truthy value, but I think this should work